content/getting-started/reactivesearch.md

```diff
@@ -78,39 +78,249 @@ new Vue({
 
 Lets add our first ReactiveSearch component: [ReactiveBase](/getting-started/reactivebase.html), it is a backend connector where we can configure the Elasticsearch index / authorization setup.
 
+We will be using `PascalCase` here. You can read more about component naming [here](https://vuejs.org/v2/guide/components-registration.html#Name-Casing).
```

Comment:
Use `kebab-case`, since it's the standard in vue, and we're using the same in whole docs.

Reply:
Done 🏁